docs/supporting-shapes/README.md

```diff
@@ -0,0 +1,191 @@
+<!--docs:
+title: "Shapes"
+layout: landing
+section: docs
+path: /docs/supporting-shapes/
+-->
+
+# Shapes
+
+The Shapes library offers a way to customize the shape of existing components, or expand your own view to have a custom shape.
+
+Currently we offer built-in functionality for shapes within our <a href="../../components/Buttons">Buttons</a> and <a href="../../components/Cards">Cards</a> components.
+
+### Shapes Library Preview
+
+Our shapes library consists of different APIs and classes you can use to build a shape. The final goal is to have your view/component consist of an object that implements the protocol `MDCShapeGenerating`. `MDCShapeGenerating` consists of one method that returns the shape’s `CGPath` for the expected size.
```

Comment:
I'm not sure what you mean by "The final goal" here - is this the final goal of our shapes library?